src/apps/stable/routes/routes.tsx

```diff
@@ -29,7 +29,7 @@ export const STABLE_APP_ROUTES: RouteObject[] = [
             },
 
             /* Public routes */
-            { index: true, loader: () => redirect('/home.html') },
+            { index: true, element: <Navigate to='/home.html' /> },
```

Comment:
Don't we need `replace` here?

Reply:
By default the `redirect` function doesn't replace. Do you want me to still replace it?

Reply:
```suggestion
            { index: true, element: <Navigate replace to='/home.html' /> },
```